test/BUILD

```diff
@@ -1,10 +1,11 @@
 cc_test(
```

Comment:
In the same spirit as my suggestion above, I encourage you to have a test target for each individual component (hello_world, scsi_defs, nvme_defs, status_translator, etc). This will allow you to run only the tests relevant to the piece you are currently working on. You can always run all of your tests with `bazel test :all`

For status_translator, this could look like this:

```
cc_library(
  name = "status_translator_test",
  srcs = ["status_translator_test.cc"],
  deps = [
      "//lib/translate:status_translator"
      "//third_party/spdk_defs:nvme_defs_lib",
      "//lib:scsi_defs_lib",
      "@googletest//:gtest_main",
      "@com_google_absl//absl/base",
  ],
)
```